code-style/javascript/README.md

```diff
@@ -26,23 +26,23 @@ Frameworks / Libraries
 
 JavaScript has a rich ecosystem of libraries. Here are some of the tools we like to use.
 
-- [React][react] - A view library for building user interfaces
-- [Vue][vue] - Another view library for building user interfaces
-- [Redux][redux] - A state management library
-- [Axios][axios] - An [isomorphic][isomorphic] HTTP client
-- [Lodash][lodash] - A utility library packed full of useful functionality
-- [Moment][moment] - A date/time library
-- [Chartist][chartist] - A responsive chart library
+- [React][react] – A view library for building user interfaces
+- [Vue][vue] – Another view library for building user interfaces
+- [Redux][redux] – A state management library
+- [Axios][axios] – An [isomorphic][isomorphic] HTTP client
+- [Lodash][lodash] – A utility library packed full of useful functionality
+- [Moment][moment] – A date/time library
+- [Chartist][chartist] – A responsive chart library
 
 Testing
 -------
 
 Automated testing is an important part of our development process. Here are some of the tools we use to test.
 
-- [Mocha][mocha] - A test runner
-- [Chai][chai] - A test assertion library
-- [Sinon][sinon] - A test mocking library
-- [Jest][jest] - Another test runner
+- [Mocha][mocha] – A test runner
+- [Chai][chai] – A test assertion library
+- [Sinon][sinon] – A test mocking library
+- [Jest][jest] – Another test runner
```

Comment:
Same as the comment above. Would it be helpful to add some more information that differentiates how we use Mocha vs Jest?

Reply:
Updated :)